messages/btc.proto

```diff
@@ -137,6 +138,9 @@ message BTCSignNextResponse {
   // Previous tx's input/output index in case of PREV_INPUT or PREV_OUTPUT, for the input at `index`.
   uint32 prev_index = 5;
   AntiKleptoSignerCommitment anti_klepto_signer_commitment = 6;
+  // Generated output. The host *must* verify its correctness using silent_payment_dleq_proof.
```

Comment:
If you write `silent_payment_dleq_proof` (with ticks `) it will be nicer formatted in the generated docs.

Reply:
Done